src/styles/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  border: 1px solid var(--gray-elements);
```

Comment:
Ensure that the SCSS variable `--gray-elements` is defined in a separate variables file and imported correctly.